### ISSUE TITLE
Semigroups no longer compiles with recent git versions of libsemigroups

### ISSUE THREAD

**Issue by wilfwilson** [gap-packages/Semigroups#680]
...because recent git versiona of libsemigroups no longer include a `.VERSION` file, and the `configure` script for Semigroups fails when it does not find it. This doesn't affect released versions of Semigroups.

(Using the current `stable-1.2` of libsemigroups and `stable-3.3` of Semigroups):

```
configure: using included libsemigroups...
checking for libsemigroups/include/libsemigroups/libsemigroups.hpp... yes
checking for libsemigroups/.VERSION... no
configure: error: libsemigroups version 1.0.6 or higher is required
```

**Comment by james-d-mitchell** [comment]
Right, I know how to fix this, an oversight, albeit a terrible one :(

**Comment by james-d-mitchell** [comment]
This was resolved by PR #693 and since the issue shouldn't impact the release version of Semigroups (since there we use the release version of libsemigroups which does include the `.VERSION` file), I'll close the issue now.
